class.FlipPage.php

```diff
@@ -397,11 +401,11 @@ protected function addAllLinks()
             $this->addLink('Logout', $this->logoutUrl);
         }
         $aboutMenu = array(
```

Comment:
changing every last hardcoded link to optionally point to localhost is my current priority
please elaborate on your idea of how to correctly and cleanly facilitate this